dev-libs/libsodium/libsodium-1.0.16.ebuild

```diff
@@ -37,7 +38,7 @@ src_configure() {
 		${myconf}
 }
 
-src_install() {
+src_multilib_install() {
 	default
```

Comment:
`default` is not valid here, it includes a possible `make` call which is undesired. Call `einstalldocs` directly instead.

Reply:
Okay, the "default" is from that example: https://wiki.gentoo.org/wiki/Project:Multilib/multilib-minimal

I've changed this now to emake DESTDIR="${D}" install, to make the deletion of *.la possible.

Reply:
Oh, I'm sorry about this, I've misread it as `src_multilib_install_all`. The `default` was correct indeed.

Reply:
Okay, but should I revert it? When using "default", the find for deleting '*.la' seems not to run for me?

Reply:
Ok, let me refocus here. Remove `multilib_src_install` = make it run the default, and do removal+docs in `_all`. Technically you could do it the other way around but then you'd unnecessarily run `find` twice.

Reply:
Okay, will do :)